client/src/components/payment/payment-mixin.ts

```diff
@@ -90,13 +89,11 @@ export default class PaymentMixin extends Vue {
       })
       await paymentModule.setPaymentFees(response)
       return true
-    } catch (error) {
-      console.error('fetchFees() =', error) // eslint-disable-line no-console
-      if (error instanceof PaymentApiError) {
-        await errorModule.setAppError({ id: 'payment-api-error', error: error.message } as ErrorI)
```

Comment:
I can see no reason to report API errors separately from other errors to the user.